views/github-create-branch.hbs

```diff
@@ -21,15 +21,61 @@
 <input type="hidden" id="_csrf" name="_csrf" value="{{csrfToken}}" />
 <input type="hidden" id="jiraHost" name="jiraHost" value="{{jiraHost}}" />
 
-{{> navigation hideBackButton=true }}
+{{> navigation }}
```

Comment:
Turns out we can just remove this completely

Reply:
Removed